docs/version-specific/update-version-specific-docs.sh

```diff
@@ -147,17 +147,23 @@ echo "* [List of available toolchain options](toolchain-opts.md)" >> $overview
 
 #  list-toolchain doc
 echo "eb --list-toolchains"
-eb --list-toolchains --output-format md | sed 's/^\(# .*\)/\1 {: #toolchains_table }/g' > eb-list-toolchains.md
-echo >> eb-list-toolchains.md
-echo "*(see also* \`\`eb --list-toolchains\`\` *)*" >> eb-list-toolchains.md
+eb --list-toolchains --output-format md | sed 's/^\(# .*\)/\1 {: #toolchains_table }/g' > toolchains.md
+echo >> toolchains.md
+echo "*(see also* \`\`eb --list-toolchains\`\` *)*" >> toolchains.md
 
-echo "* [List of known toolchains](eb-list-toolchains.md)" >> $overview
+echo "* [List of known toolchains](toolchains.md)" >> $overview
```

Comment:
In #96 we renamed `eb-list-toolchains.md` to `toolchains.md` manually, but the script overwrites this again, so needs to be changed here too